get-new-tool-versions/get-new-tool-versions.ps1

```diff
@@ -29,6 +29,7 @@ if ($ToolName -eq "Xamarin") {
 if ($VersionsToBuild) {
     $availableVersions = $VersionsToBuild -join $joinChars
     Write-Host "The following versions are available to build:`n${availableVersions}"
+    Write-Host "::set-output name=version_number::${availableVersions}"
     Write-Host "##vso[task.setvariable variable=TOOL_VERSIONS;isOutput=true]${availableVersions}"
```

Comment:
In my opinion, the name of the `version_number` variable does not match its value. Can we name it `TOOL_VERSIONS` for consistency

Reply:
Renamed this variable